Dockerfile

```diff
@@ -10,9 +10,9 @@
 #
 # http://github.com/jhowardmsft/busybox
 
-FROM microsoft/windowsservercore
+FROM mcr.microsoft.com/windows/servercore:ltsc2022
```

Comment:
Do you know if the previous images were multi-arch (so `microsoft/windowsservercore` would allow picking the correct variant?

Reply:
They were not (and the new ones are not either: https://explore.ggcr.dev/?image=mcr.microsoft.com/windows/servercore:ltsc2022)

Reply:
(disagreeing with that decision is actually the original reason I created https://explore.ggcr.dev/?image=hell/win:core :joy:)